data/kahet/kahet missions.txt

```diff
@@ -17,6 +17,7 @@ mission "Graveyard Label"
 	source
 		attributes ka'het
 	on offer
+		set "global: First Contact: Ka'het"
```

Comment:
I don't think this is the best spot for this condition, since this mission offers anywhere in the Graveyard that you land, which may be done without ever seeing any Ka'het ships.

Reply:
Possible? Yes, likely? No. I have never once been in the graveyard long enough to actually land on a planet without seeing a Ka'het. Half the time I can't even mistakenly come through the wormhole, turn around, and go back in without seeing one.

In any case, they made it to the Builder/Ka'het worlds, and that is the entire purpose of these "First Contact" variables. To be used by achievements and other checks to ensure that the player has reached this stage in the game.